gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -126,6 +127,7 @@ public final class InstantiatingGrpcChannelProvider implements TransportChannelP
   @Nullable private final Boolean allowNonDefaultServiceAccount;
   @VisibleForTesting final ImmutableMap<String, ?> directPathServiceConfig;
   @Nullable private final MtlsProvider mtlsProvider;
+  @Nullable private final ArrayList<String> allowedHardBoundTokenTypes;
```

Comment:
Please use `List` instead of `ArrayList`.

Reply:
Done in https://github.com/googleapis/sdk-platform-java/pull/3467/commits/e193cc9412ae68b01ca5d1a21eb38baf35d68a71